### DISCUSSION TITLE
How to reorder layers i Qfield app?

### DISCUSSION THREAD

**Discussion by Batn00b** [opengisch/QField#4365] [Q&A]
Hello!
Maybe I am stupid, but i cannot find a way to reorder the layers in the Qfield app on android. I suppose it must me possble - but how.
Please help!
thx

**Comment by VxTedxV** [comment]
It's not. At first, QField is intended to reflect what is done in QGis. Then, some additional functionalities enable the ability to show/hide layers for example (long press on layer name to access these functionalities).
